.github/workflows/release_nightly.yml

```diff
@@ -108,18 +108,27 @@ jobs:
             echo "Server terminated successfully"
           fi
 
-      - name: Publish to PyPI
-        env:
-          POETRY_PYPI_TOKEN_PYPI: ${{ secrets.PYPI_API_TOKEN }}
-          UV_PUBLISH_TOKEN: ${{ secrets.PYPI_API_TOKEN }}
+      - name: Build Docker Image
```

Comment:
the test-docker workflow should do this build

Reply:
Did you mean that I need to build the same tag in docker_test? Or did you mean, should I build the Docker just like it is in docker_test?